lib/errors.js

```diff
@@ -38,9 +38,23 @@ util.inherits(error.BaseError, Error);
  * @constructor
  */
 error.ValidationError = function(message, errors) {
+  var self = this;
+
   error.BaseError.apply(this, arguments);
   this.name = 'SequelizeValidationError';
   this.errors = errors || [];
+
+  if (this.errors.length > 0 && this.errors[0].message) {
+    this.message = "";
+
+    this.errors.forEach(function(_err, i) {
```

Comment:
True, I hadn't considered that :)

```js
this.message = this.errors.map(function (err) {
  return err.message;
}).join('; ');
```

I just really want to avoid that ugly `if i > 0` thing